packages/browser/src/core/analytics/index.ts

```diff
@@ -127,6 +135,8 @@ export class Analytics
           : options?.group,
         cookieOptions
       ).load()
+
+    this._storage = new UniversalStorage(disablePersistance, cookieOptions)
```

Comment:
I feel like it might be better adherence to the  Interface Seg. Principal to inject this via constructor for max flexibility  -- especially if we allow you to pass any storage (I could see this being a cojfig option)